### PR TITLE
Add meta map and update process label to mitohifi/findmitoreference

### PR DESCRIPTION
**Pull request by mahesh-panchal**
Adds meta map to mitohifi/findmitoreference.

## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/mitohifi/findmitoreference/environment.yml
Comment (SPPearce) [inline modules/nf-core/mitohifi/findmitoreference/environment.yml:1]: Does this tool not work with conda?

**Review by SPPearce — APPROVED** [review]
Looks fine.
Is there a possibility that users might want to rename the output files from the default names via a prefix?

File: modules/nf-core/mitohifi/findmitoreference/environment.yml
Comment (mahesh-panchal) [reply modules/nf-core/mitohifi/findmitoreference/environment.yml:1]: No, there's no conda package for it.

**Comment by mahesh-panchal** [comment]
I think in this case, it's finding the closest reference to the input sample, so it shouldn't be named after the sample.

**Pull request merged**